React/todoApp/src/app/store.js

```diff
@@ -0,0 +1,35 @@
+import { configureStore } from "@reduxjs/toolkit";
+import todosReducer from "../features/todos/todosSlice";
+
+function loadFromStorage() {
+  try {
+    const raw = localStorage.getItem("todos_state");
+
+    if (!raw) {
+      return undefined;
+    }
+
+    return JSON.parse(raw);
+  } catch {
+    return undefined;
```

Comment:
We can throw an error here. The best thing would be to use a toast library to show a toast